src/sql/azdata.d.ts

```diff
@@ -4172,15 +4186,6 @@ declare module 'azdata' {
 		operation: (operation: BackgroundOperation) => void;
 	}
 
-	namespace tasks {
```

Comment:
Where did the methods you removed go?

Reply:
They moved - we had duplicate namespace definitions so I just merged them into a single one.

Reply:
So like this one moved here for example :

https://github.com/microsoft/azuredatastudio/pull/8931/files#diff-aae8d05d08afb282d07b39695b013887R2528